functions/Get-SqlIndexes.ps1

```diff
@@ -0,0 +1,107 @@
+﻿Function Get-SqlIndexes
+{
+<#
+.SYNOPSIS
+Get a list of indexes
+
+.DESCRIPTION
+This command returns the indexes of a database or of a list of databases. 
+
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.NOTES
+Tags: Maintenance
+Author: Steffen Kampmann: https://github.com/abbgrade
+
+dbatools PowerShell module (https://dbatools.io)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.
+
+.LINK
+ https://dbatools.io/Get-SqlIndexes
+
+.EXAMPLE
+Get-SqlIndexes.ps1 -SqlServer "(localdb)\MSSQLLocalDB", "(localdb)\v11.0"
+Returns all indexes for localdb instances MSSQLLocalDB and v11.0.
+
+#>
+	[CmdletBinding()]
+	Param (
+		[Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlInstance")]
+		[string[]]$SqlServer
```

Comment:
This would have been [object[]]$SqlServer since we'll want to pass SMO objects in the future (We need to fix that in some older commands), and also the $SqlCredential object